spec-0002/index.md

```diff
@@ -3,50 +3,70 @@ title: "SPEC 2 — API Dispatch"
 number: 2
 date: 2021-12-16
 author:
-  - "Ivan Yashchuk <ivan.yashchuk@quansight.com>"
-  - "Ralf Gommers <rgommers@quansight.com>"
   - "Jarrod Millman <millman@berkeley.edu>"
-  - "Stéfan van der Walt <stefanv@berkeley.edu>"
-discussion: https://discuss.scientific-python.org/t/spec-2-api-dispatch/173
+discussion:
 endorsed-by:
 ---
 
 ## Description
 
 <!--
 Briefly and clearly describe the recommendation.
+-->
+
+We propose mechanisms for wholesale reimplementations of library functions.
+This would allow groups outside of, say, `networkx` to (a) provide new
+functions to replace parts of `networkx`, or (b) provide data structures
+that can pass through NetworkX's existing computational pipelines.
+
+This SPEC focuses on the rationale for these mechanisms, and provides
+links to implementations related technical discussions.
+
+### Core Project Endorsement
+
+<!--
+Briefly discuss what it means for a core project to endorse this SPEC.
+-->
+
+### Ecosystem Adoption
+
+<!--
+Briefly discuss what it means for a project to adopt this SPEC.
+-->
```

Comment:
Non blocking comment: we were discussing that maybe these don't have to be in the skeleton. Endorsement and adoption is defined globally for all SPECs and should mostly be the same. Hence, unless there is a distinction specific to this SPEC which needs to be made, I would remove these sections.

Reply:
@tupui Do you mind filing this as an issue for discussion?